src/views/Register.vue

```diff
@@ -45,10 +46,18 @@ export default {
       email: "",
       password: "",
       username: "",
+      passwordConfirm: "",
+      rules: {
+        samePass: v => v === this.password || 'Passwords do not match'
+      }
     };
   },
   methods: {
     register() {
+      if (this.password != this.passwordConfirm){
```

Comment:
Since this is a password, should we use !== instead to have a strict type match as well?

Reply:
Fixed!